spec/mongoid/tasks/models.rb

```diff
@@ -0,0 +1,13 @@
+module DatabaseSpec
+  class Measurement
+    include Mongoid::Document
+
+    field :timestamp, type: Time
+    field :temperature, type: Integer
+
+    store_in collection: "measurement",
+      collection_options: {
+        capped: true, size: 10000
+      }
```

Comment:
Is there a test for time series options?

Reply:
No, I do not think we should test all the possible options in Mongoid. I use capped collection options because they are accepted by most server versions.